07_errors/patrickmarabeas/types.go

```diff
@@ -1,21 +1,19 @@
 package main
 
-import (
-	"sync"
-)
+import "sync"
 
 type Puppy struct {
 	ID    int
 	Breed string
 	Color string
-	Value string
+	Value int // cents
```

Comment:
Two ways you could convey this information without a comment:
1) Change the field name to `ValueCents`
2) Add a new type: `type Cents int`

Just a thought.